### ISSUE TITLE
Arkos 1.6 - issues

### ISSUE THREAD

**Issue by skelph** [krishenriksen/AnberPorts#3]
Gforce now don't work
Stadia don't work
Moonlight embedded don't work
Anbercloud etc etc etc ....

Just a flashing cursor

**Comment by skelph** [comment]
tried installing deleting reinstalling arkos cant get anywhere with moonlight any hints

**Comment by RetroBreak** [comment]
I'm having the same problem. [https://www.reddit.com/r/RG351/comments/mm0gsn/cant_get_moonlight_streaming_to_work_gets_stuck/](url)

**Comment by Flexcast8** [comment]
I am having the same issue. Moonlight opens to a black screen and that's it. I've tried reinstalling ark os and anberports.

**Comment by elodlukacs** [comment]
Same here.

**Comment by grumpyoleg** [comment]
Same here on arkos 1.5 and 1.6.

**Comment by skelph** [comment]
[log.txt](https://github.com/krishenriksen/AnberPorts/files/6385255/log.txt)
nope seems to be a file permission thing

**Comment by krishenriksen** [comment]
> [log.txt](https://github.com/krishenriksen/AnberPorts/files/6385255/log.txt)
> nope seems to be a file permission thing

what are you trying to do here?

**Comment by krishenriksen** [comment]
Moonlight has been fixed.

**Comment by skelph** [comment]
Yes got it working screen flashes a lot tho any luck on stadia / gforce now ???

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by krishenriksen** [comment]
stadia / gforce will come in AnberX, keep an eye out for https://github.com/krishenriksen/AnberX

**Comment by skelph** [comment]
Thanks buddy thanks for the great work got moonlight working

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
